week6/README.md

```diff
@@ -0,0 +1,42 @@
+# Week 6 Homework
+
+## Part 1 - Short Answer Questions
+
+### a) In SwiftUI, you describe your interfaces _____________ and leave the ___________ to the Framework.
+- Declaratively; rendering.
+
+### b) When a View first appears, and you want it to animate in, you can do so in the Views ___________ modifier.
+- `.onAppear`.
+
+### c) In Swift, the native types ______________ and ____________ implement the Hashable protocol.
+- `String`; `Int`.
+
+### d) ____________ is how long an animation takes to complete.
+- Duration.
+
+### e) The ScrollViewReader requires each view inside the ScrollView to have a unique ___________ to identify and navigate to them.
+- `id`.
+
+### f) ____________ protocol’s only requirement is to have an id property that conforms to Hashable.
+- `Identifiable`.
+
+### g) A ____________ Grid lets you specify an exact size for a column or row.
+- Fixed-size.
+
+### h) ________________ allows you to scroll to any position inside a ScrollView programmatically.
+- `ScrollViewReader`.
+
+### i) True or False: All animations in SwiftUI are interruptible and reversible by default.
+- True.
+
+### j) The _________ defines how items in a Grid should be sized and aligned.
+- `Layout`.
+
+### k) Wrap any changes you want to animate in a call to  _________________
+- `.animation()`.
+
+### l) The native SwiftUI grid view builds on the _____________ and _____________ views.
+- `LazyVGrid`; `LazyHGrid`.
```

Comment:
A more precise answer would be `VStack` and `HStack`. Those are the basic building blocks of a Grid.